packages/commonwealth/globals.d.ts

```diff
@@ -1,17 +1,3 @@
-declare module '*.scss';
-
-/*
-declare namespace JSX {
-  interface ElementClass {
-    view: any;
-  }
-
-  interface IntrinsicElements {
-    __props: any;
-  }
-
-  interface ElementAttributesProperty {
-    __props: any;
-  }
-}
-*/
\ No newline at end of file
+declare module "*.png"
+declare module "*.svg"
+declare module "*.module.scss"
```

Comment:
would we have to include jpeg definition if we we are to use jpegs?

Reply:
I think this file can actually be removed since vite has their own overrides for the non-typescript import types.